src/rules/ban_types.rs

```diff
@@ -31,6 +31,44 @@ impl LintRule for BanTypes {
     let mut visitor = BanTypesVisitor::new(context);
     visitor.visit_module(module, module);
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Bans the use of primitive wrapper objects (e.g. `String` the object is a 
+      wrapper of `string` the primitive) in addition to the non-explicit `Function`
+      type and the misunderstood `Object` type. 
+
+There are very few situations where primitive wrapper objects are desired and
```

Comment:
```suggestion
  fn docs(&self) -> &'static str {
    r#"Bans the use of primitive wrapper objects (e.g. `String` the object is a 
wrapper of `string` the primitive) in addition to the non-explicit `Function`
type and the misunderstood `Object` type. 

There are very few situations where primitive wrapper objects are desired and
```

I'm not sure, but I feel like leading spaces would result in being rendered a little badly.

Reply:
Markdown should automatically handle extra spaces gracefully (by stripping them).  Regardless, your suggestion is neater and my extra spaces unintentional.